docker/prodlike/config/signer-config.toml

```diff
@@ -0,0 +1,261 @@
+[bitcoin]
+# The URI(s) of the Bitcoin Core RPC server(s) to connect to.
+#
+# You may specify multiple Bitcoin Core RPC servers if you have them. They will
+# be randomly tried until one succeeds.
+#
+# Format: ["http://<user>:<pass>@<host>:<port>", ..]
+# Default: <none>
+# Required: true
+# Environment: SIGNER_BITCOIN__RPC_ENDPOINTS
+# Environment Example: http://user:pass@seed-1:4122,http://foo:bar@seed-2:4122
+rpc_endpoints = [
+    "http://user:pass@your-bitcoin-node:4122",
+]
+
+# The URI(s) of the Bitcoin Core ZMQ block hash stream(s) to connect to.
+#
+# You may optionally specify multiple endpoints if you have them. They will be
+# tried in order until one succeeds, and it will attempt failover to the next
+# endpoint if the connection is lost.
+#
+# Format: ["tcp://<host>:<port>", ..]
+# Default: <none>
+# Required: true
+# Environment: SIGNER_BITCOIN__BLOCK_HASH_STREAM_ENDPOINTS
+# Environment Example: tcp://10.0.0.1:28332,tcp://10.0.0.2:28332
+block_hash_stream_endpoints = [
+    "tcp://localhost:28332"
+]
+
+# !! ==============================================================================
+# !! Block Notifier Configuration
+# !! ==============================================================================
+# Electrum server connection confirmation.
+[block_notifier]
+# The URI of the Electrum server to connect to.
+#
+# Format: "<protocol>://<host>:<port>"
+# Default: <none>
+# Required: true
+# Environment: SIGNER_BLOCK_NOTIFIER__SERVER
+server = "tcp://localhost:60401"
+retry_interval = 10
+max_retry_attempts = 5
+ping_interval = 60
+subscribe_interval = 10
+
+# !! ==============================================================================
+# !! Stacks Node Configuration
+# !! ==============================================================================
+[stacks]
+# The RPC URL(s) of the Stacks node(s) to connect to. At least one must be
+# provided. If multiple nodes are provided they will be tried in order when
+# making requests.
+endpoints = ["stacks-node-rpc"]
+
+# This is the start height of the first EPOCH 3.0 block on the stacks
+# blockchain.
+nakamoto_start_height = 867867
+
+# !! ==============================================================================
+# !! Signer Configuration
+# !! ==============================================================================
+[signer]
+# The private key associated with the signer. This is used to generate the
+# signers associated public key and sign messages to other signers.
+#
+# This may be either in 32- or 33-byte format. If you generated the key using
+# `stacks-cli` or other ecosystem tools, it is likely that the key is in 33-byte
+# format which includes a stacks-proprietary suffix byte. The sBTC signer doesn't
+# make use of this byte and it will be trimmed automatically if provided.
+#
+# Format: "<hex-encoded-private-key>" (64 or 66 hex-characters)
+# Required: true
+# Environment: SIGNER_SIGNER__PRIVATE_KEY
+private_key = "your-private-key"
```

Comment:
I'd change this to `"0000000000000000000000000000000000000000000000000000000000000001"
`

Reply:
Done.